src/lib/viewers/controls/media/MediaSettingsControls/MediaSettingsRadioItem.scss

```diff
@@ -0,0 +1,53 @@
+@import './styles';
+
+.bp-MediaSettingsRadioItem {
+    @include bp-MediaSettingsRow;
+
+    &.bp-is-selected {
+        .bp-MediaSettingsRadioItem-check-icon {
+            visibility: visible;
+        }
+
+        .bp-MediaSettingsRadioItem-value {
+            color: $box-blue;
+        }
+    }
+
+    &:hover {
+        background-color: $hover-blue-background;
+
+        .bp-MediaSettingsRadioItem-check-icon {
+            fill: $blue-steel;
+        }
+
+        .bp-MediaSettingsRadioItem-value {
+            color: $blue-steel;
+        }
+    }
+
+    .bp-is-focused &:focus {
+        background-color: $box-blue;
+
+        .bp-MediaSettingsRadioItem-check-icon {
+            fill: $white;
+        }
+
+        .bp-MediaSettingsRadioItem-value {
+            color: $white;
+        }
+    }
+}
+
+.bp-MediaSettingsRadioItem-check {
+    @include bp-MediaSettingsRow-cell;
+}
+
+.bp-MediaSettingsRadioItem-check-icon {
+    visibility: hidden;
+}
+
+.bp-MediaSettingsRadioItem-value {
+    @include bp-MediaSettingsRow-value;
+
+    text-align: center;
```

Comment:
Is this intentional? Seems like the  values are left aligned currently

Reply:
It matches the experience in production, though I don't think it has much practical impact.

Reply:
Actually, I'm mistaken. The text-align: center is on the "back" component. Great catch.